cluster/mesos/docker/docker-compose.yml

```diff
@@ -30,6 +30,7 @@ mesosslave:
   hostname: mesosslave
   privileged: true
   image: mesosphere/mesos-slave-dind:mesos-0.24.0_dind-0.2_docker-1.8.2_ubuntu-14.04.3
+  ports: [ "10248","10249" ]
```

Comment:
I exposed these so that I could curl the /debug/pprof/... endpoints. Docker
publishes these ports for me, but they're actually a bit flakey when I try
to access them via curl from the host. Thoughts? I'm probably not
understanding all of the network magic here.

On Fri, Oct 16, 2015 at 1:40 AM, Dr. Stefan Schimanski <
notifications@github.com> wrote:

> In cluster/mesos/docker/docker-compose.yml
> <https://github.com/kubernetes/kubernetes/pull/15735#discussion_r42210107>
> :
>
> > @@ -30,6 +30,7 @@ mesosslave:
> >    hostname: mesosslave
> >    privileged: true
> >    image: mesosphere/mesos-slave-dind:mesos-0.24.0_dind-0.2_docker-1.8.2_ubuntu-14.04.3
> > +  ports: [ "10248","10249" ]
>
> Do you know really need these port forwarding? Elsewhere we depend on
> proper routes to the container IPs.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/15735/files#r42210107>.
>